KMB/make_KMB_info.py

```diff
@@ -8,13 +8,17 @@
 """
 from __future__ import unicode_literals
 from collections import OrderedDict
+import os.path
 
 import pywikibot
+from pywikibot.data import sparql
+
 import batchupload.helpers as helpers
 import batchupload.common as common
 from batchupload.make_info import MakeBaseInfo
 
 
+MAPPINGS_DIR = 'mappings'
 BATCH_CAT = 'Media contributed by RAÄ'  # stem for maintenance categories
```

Comment:
We have both `Images from the Swedish National Heritage Board` and `... by RAÄ`? Let's keep the names consistent to avoid confusing people even more :)